test/java/org/stathissideris/ascii2image/test/TextGridTest.java

```diff
@@ -227,7 +227,7 @@
 	@Test public void testFindBoundariesExpandingFromUOutside() throws FileNotFoundException, IOException {
 		TextGrid grid;
 		grid = new TextGrid();
-		grid.loadFrom("tests/text/simple_U01.txt");
+		grid.loadFrom("test-resources/text/simple_U01.txt");
 
```

Comment:
After introducing ```lein-junit```, which executes JUnit based tests under tests directory, I realized existing tests are not passing since we do not have resources under ```tests``` directory but in ```test-resources``` directory.
These changes will make those tests work again.